pkg/api/aim/dashboards.go

```diff
@@ -5,16 +5,27 @@ import (
 
 	"github.com/gofiber/fiber/v2"
 	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
 	"gorm.io/gorm"
 
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api"
+	"github.com/G-Research/fasttrackml/pkg/common/middleware/namespace"
 	"github.com/G-Research/fasttrackml/pkg/database"
 )
 
 func GetDashboards(c *fiber.Ctx) error {
+	ns, err := namespace.GetNamespaceFromContext(c.Context())
```

Comment:
Does golang/fiber provide any sort of before hook that would let us DRY this up?

Reply:
no. this is how we can attach `middleware`.

Reply:
no, this is usual approach to get something from the context.

Reply:
it could be DRY'd up if we moved the "fetch from context" step down to the repository